Bugzilla/Elastic/Indexer.pm

```diff
@@ -181,6 +181,11 @@ sub bulk_load {
 
     $self->_bulk_load_ids($bulk, $class, $new_ids) if @$new_ids;
     $self->_bulk_load_ids($bulk, $class, $updated_ids) if @$updated_ids;
+
+    return {
```

Comment:
Consider adding `class => $class` here, since it's free information for you to provide but impossible to distinguish from the results alone otherwise. (See also various review thoughts below about closer stats integration here.)